ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateTransition.java

```diff
@@ -105,12 +109,11 @@ private void blockProcessor(BeaconStateWithCache state, BeaconBlock block) {
       BlockProcessorUtil.process_block_header(state, block);
       BlockProcessorUtil.process_randao(state, block);
       BlockProcessorUtil.process_eth1_data(state, block);
-      BlockProcessorUtil.process_proposer_slashings(state, block);
-      BlockProcessorUtil.process_attester_slashings(state, block);
-      BlockProcessorUtil.process_attestations(state, block);
-      BlockProcessorUtil.process_deposits(state, block);
-      BlockProcessorUtil.process_voluntary_exits(state, block);
-      BlockProcessorUtil.process_transfers(state, block);
+      BlockProcessorUtil.process_proposer_slashing(state, block);
```

Comment:
I had made these process_proposer_slashings, process_attester_slashings and process_attestations plural because they take in the whole block, which is different than the functions in spec.

Reply:
if you could just revert all of these 6 lines of change, that would be amazing.

Reply:
Ok cool - will do!